modules/network-observability-SRIOV-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// network_observability/configuring-operator.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-SR-IOV-config_{context}"]
+= Configuring the monitoring of SR-IOV interface traffic
+In order to collect traffic from a cluster with Single Root I/O Virtualization (SR-IOV) device, the `FlowCollector` `spec.agent.ebpf.privileged` specification value must be `true`. Then, the eBPF agent monitors other network namespaces in addition to the host network namespaces, which are monitored by default. When a pod with a virtual functions (VF) interface is created, a new network namespace is created. With `SRIOVNetwork` policy `IPAM` configurations specified, the VF interface is migrated from the host network namespace to the pod network namespace.
+
+.Prerequisite
+* Access to an {product-title} cluster with SR-IOV device. 
+* The `SRIOVNetwork` custom resource (CR) specifies `IPAM` configured with an IP address from the range in which interface has the IP or from other plugins.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. 
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A sample configuration is as follows:
++
+[id="network-observability-flowcollector-configuring-SRIOV-monitoring{context}"]
+.Configure `FlowCollector` for packet drops
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: DIRECT
+  agent:
+    type: EBPF
+    ebpf:        
+      privileged: true             <1> 
+----
+<1> Note that the `spec.agent.ebpf.privileged` specification value must be `true` for packet drop tracking to be enabled.
```

Comment:
* I don't think the "Note that" is necessary
* Same comment about specification value

```suggestion
<1> The `spec.agent.ebpf.privileged` field value must be `true` to enable packet drop tracking.
```